.nvmrc

```diff
@@ -0,0 +1 @@
+6.9.5
```

Comment:
Ahh okay, we should probably update [quilt](https://github.com/Shopify/quilt/blob/master/.nvmrc#L1) then. That's what I used as a reference.